Formula/openmodelica.rb

```diff
@@ -0,0 +1,72 @@
+class Openmodelica < Formula
+  desc "Open-source modeling and simulation tool"
+  homepage "https://openmodelica.org/"
+  # GitHub's archives lack submodules, must pull:
+  url "https://github.com/OpenModelica/OpenModelica.git",
+    tag:      "v1.16.5",
+    revision: "11fcab4f2d6895f2db073572b2bff1a43177313f"
+  license "GPL-3.0-only"
+  head "https://github.com/OpenModelica/OpenModelica.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build # for gfortran
+  depends_on "gnu-sed" => :build
+  depends_on "libtool" => :build
+  depends_on "openjdk" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "gettext"
+  depends_on "hdf5"
+  depends_on "hwloc"
+  depends_on "lp_solve"
+  depends_on "omniorb"
+  depends_on "openblas"
+  depends_on "qt@5"
+  depends_on "readline"
+  depends_on "sundials"
+
+  uses_from_macos "curl"
+  uses_from_macos "expat"
+  uses_from_macos "libffi"
+  uses_from_macos "ncurses"
+
+  def install
+    ENV.append_to_cflags "-I#{MacOS.sdk_path_if_needed}/usr/include/ffi"
+    args = %W[
+      --prefix=#{prefix}
+      --disable-debug
+      --disable-modelica3d
+      --with-cppruntime
+      --with-hwloc
+      --with-lapack=-lopenblas
+      --with-omlibrary=core
+      --with-omniORB
+    ]
+
+    system "autoconf"
+    system "./configure", *args
+    # omplot needs qt & OpenModelica #7240.
+    # omparser needs OpenModelica #7247
+    # omshell, omedit, omnotebook, omoptim need QTWebKit: #19391 & #19438
+    # omsens_qt fails with: "OMSens_Qt is not supported on MacOS"
+    system "make", "omc", "omlibrary-core", "omsimulator"
+    prefix.install Dir["build/*"]
+  end
+
+  test do
+    system "#{bin}/omc", "--version"
+    # assert_match "OMPlot", shell_output("#{bin}/OMPlot --help", 1) # help is the only nonblocking flag
```

Comment:
```suggestion
    assert_match "OMPlot", shell_output("#{bin}/OMPlot --help 2>&1", 1) # help is the only nonblocking flag
```
If the `assert_match` isn't working then you may just need to redirect stderr.

Reply:
Oh no it prints to STDOUT via `fprintf`, it's just commented out bc OMPlot itself isn't enabled.  I can remove it if needed?

Reply:
I see; just remove it then, if we don't build OMPlot.

Reply:
done, typo in `omc` `shell_output` test fixed (🤦‍♂️🤦‍♀️🤦‍♂️🤦‍♀️), I think this push is the one